suite-native/navigation/src/components/ScreenHeaderWithIcons.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode } from 'react';
+import { View } from 'react-native';
+
+import { NativeStyleObject, prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+type ScreenHeaderWithIconsProps = {
+    leftIcon?: ReactNode;
+    rightIcon?: ReactNode;
+    children: ReactNode;
+    style?: NativeStyleObject;
+};
+
+const ICON_SIZE = 48;
+
+const screenHeaderStyle = prepareNativeStyle(() => ({
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    height: ICON_SIZE,
+}));
+
+const iconWrapperStyle = prepareNativeStyle(() => ({
+    width: ICON_SIZE,
```

Comment:
I would probably prefer solution with position relative and predefined height only since it is more versatile, but no big deal. Because it's single component now it could be easily refactored if this solution won't work well... I will leave decision to you.